app/views/courses/preview/_entry_requirements_qualifications.html.erb

```diff
@@ -13,9 +12,13 @@
     </aside>
 
     <h3 class="govuk-heading-m">Qualifications needed</h3>
-    <div data-qa="course__required_qualifications">
-      <%= markdown(course.required_qualifications) %>
-    </div>
+    <% if course.provider.recruitment_cycle_year.to_i >= 2022 %>
```

Comment:
There is a number of recent introduction of `2022` checking.

Can we set that ubiquitously across either or both codebases.
ie

```ruby
course.provider.recruitment_cycle_year.to_i >= CHANGES_INTRODUCED_IN_2022
```

Reply:
Yea that 100% makes sense. Will do